app/assets/stylesheets/components/_lookup.scss

```diff
@@ -0,0 +1,5 @@
+.app-c-lookup {
```

Comment:
This isn't a component, so it shouldn't use the `app-c` prefix for class names. It should still use BEM, so maybe something like `postcode-lookup` would be good. These styles should also be moved out of the components directory and put in `finder_frontend.scss` instead.

Reply:
All makes sense as it's a partial